apps/web/components/Charts/ChartsCard/ChartsCard.tsx

```diff
@@ -56,34 +56,15 @@ export const ChartsCard: React.FC<ChartsCardsProps> = ({ data, blue }) => {
   }
 
   const items = (
-    <Box
-      ref={ref}
-      display="flex"
-      style={{ flexDirection: 'column' }}
-      flexGrow={1}
-      flexDirection={shouldStack ? 'columnReverse' : 'row'}
-      alignItems="stretch"
-      justifyContent="flexStart"
-    >
+    <Box ref={ref} className={cn(styles.card)}>
       <Box
+        className={cn(styles.outerWrapper)}
+        background={subPage ? 'blue100' : 'purple100'}
         style={{
-          width: '100%',
-          minHeight: '156px',
-          borderTopLeftRadius: '8px',
-          borderTopRightRadius: '8px',
+          width: graph.type === 'Pie' ? '100%' : '889px',
```

Comment:
we should try to avoid inline styles, in this case it looks like outerWrapper style has width 100% that would always be overwritten.
Probably better to define className for graph variants if they differ